HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1516,7 +1506,7 @@ def create_schedule_file(col_name:,
     schedule_length = @schedules[col_name].length
     min_per_item = 60.0 / (schedule_length / num_hrs_in_year)
 
-    schedule_file = OpenStudio::Model::ScheduleFile.new(@external_file)
+    schedule_file = OpenStudio::Model::ScheduleFile.new(@model, OpenStudio::Path.new(@output_schedules_path))
```

Comment:
We didn't actually implement the `filename` -> `filepath` change in the ctor. (Why did we do a ci skip on this commit?)

Reply:
I'm not sure what your first comment means, can you clarify? 

We skipped the CI because it wasn't in a docker build at the time; now that it's been merged, we'll be able to develop and run the tests. Thanks, I had forgotten about that.

Reply:
Oh, I forgot I changed the ctor to accept a `filepath` instead of a `filename`. I was confused because the OS-resources test (which still uses `filename`) wasn't showing a test failure...

Reply:
Ok, tests passing. Probably shouldn’t merge this in until rc2 is out though.

Reply:
I'm showing that you don't _need_ to call `OpenStudio::Path.new on `@output_schedules_path`, but I'm not sure why.